src/main/java/com/ldtteam/structurize/client/BlueprintRenderer.java

```diff
@@ -194,6 +195,7 @@ private void init(final BlueprintPreviewData previewData, final Map<Object, Exce
         }
 
         blockAccess.setSolidSubstitutionOverride(null);
+        blockAccess.setRenderBlocksNiceOverride(Structurize.getConfig().getClient().renderPlaceholdersNice.get());
 
```

Comment:
I think here too

Reply:
That is actually the same line Nightenom commented about earlier :D

Reply:
So we have a client config not null check here before this?

Reply:
ahhh